samples/graph-activity-feed/csharp/ActivityFeedNotification/Views/Home/Details.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿
+@model IList<TabActivityFeed.Model.TaskInfo>
```

Comment:
Check if this is used, if not please delete.

Reply:
Yes this is used as partial view